pkg/metrics/metrics.go

```diff
@@ -0,0 +1,70 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	buildCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "build_builds_total",
+			Help: "Number of total created Builds.",
+		},
+		[]string{"build_buildstrategy"})
```

Comment:
I suggest to use constants for the label keys as they are used for multiple metrics.

Reply:
Good suggestion! If we still need this PR, I can change that with the naming improvement in your today's first review.

Thanks!

Reply:
Refine by using:
```
const (
	buildStrategyLabel string = "buildstrategy"
	namespaceLabel string = "namespace"
)
```